crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -18,15 +18,17 @@
 
 module Api
   class Crowbar < Tableless
-    attr_reader :version
+    attr_reader :version, :addons
```

Comment:
[I already queried this](https://github.com/crowbar/crowbar-core/pull/674/commits/72a52c584d5a3f12fd93a322793b4a97ed6639e6#r80055610) in the PR from which this is backporting stuff.  What's the point of having both `#addons` and `#features`, given that they return the same value?  The downside is a more confusing interface, and the only benefit I can see is that `@addons` memoizes the result of `#features`, so that it only executes it once, but then why not memoize on demand, instead of in the constructor, so that `#features` is only called if it's really needed?